stats/lib/stats-fetchers.js

```diff
@@ -148,6 +148,20 @@ export const fetchParticipantScheduledRewards = async (pgPools, { from, to }, ad
   return rows
 }
 
+/**
+ * @param {PgPools} pgPools
+ * @param {import('./typings.js').DateRangeFilter} filter
+ * @param {string} address
+ */
+export const fetchParticipantRewardTransfers = async (pgPools, { from, to }, address) => {
+  const { rows } = await pgPools.stats.query(`
+    SELECT day::TEXT, amount
+    FROM daily_reward_transfers
+    WHERE to_address = $1 AND day >= $2 AND day <= $3
```

Comment:
Could you please check that we have an SQL index to make this query performant? I think the index should be over `to_address` first and `day` second.

Reply:
We have these:

```sql
CREATE INDEX daily_reward_transfers_day ON daily_reward_transfers (day);
CREATE INDEX daily_reward_transfers_last_block ON daily_reward_transfers (last_checked_block DESC);
```

Will add